framework/src/main/java/org/apache/drill/test/framework/ColumnList.java

```diff
@@ -161,12 +161,17 @@ private boolean compare(ColumnList o1, ColumnList o2) {
           }
           break;
         case Types.DOUBLE:
-          double d1 = (Double) list1.get(i);
-          double d2 = (Double) list2.get(i);
-          if ((d1 + d2) / 2 != 0) {
-            if (!(Math.abs((d1 - d2) / ((d1 + d2) / 2)) < 1.0E-12)) return false;
-          } else if (d1 != 0) {
-            return false;
+          Double d1 = (Double) list1.get(i);
```

Comment:
For the test framework changes, can you please open a separate PR?

Reply:
sure, will do.

Reply:
The changes have been rolled back, new pull request - https://github.com/mapr/drill-test-framework/pull/420 , could you please review?